bucket/emacs-kl.json

```
@@ -8,7 +8,10 @@
     "installer": {
         "script": [
             "Get-ChildItem \"$dir\\*.msix\" | %{Expand-7zipArchive $_.FullName \"$dir\"}",
-            "Remove-Item -Force \"$dir\\*.msix\""
+            "Remove-Item -Force \"$dir\\*.msix\"",
+            "Push-Location \"$dir\"",
+            "Rename-Item 'bin\\libstdc%2B%2B-6.dll' 'libstdc++-6.dll'",

```

There're several files affect by the encoded name.
Can you help to rename them all?
Something like
```suggestion
            "dir 'bin\\*%2b*' | %{ren \"$_\" \"$($_.Name -replace '%2b','+')\"",
```